WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewController.swift

```diff
@@ -12,6 +12,12 @@ final class HubMenuViewController: UIHostingController<HubMenu> {
         configureTabBarItem()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        // Workaround on iOS 16: hide the navigation bar before showing the hub menu
+        navigationController?.isNavigationBarHidden = true
```

Comment:
nit: does it make sense to add an iOS 16+ condition so that we know we can replace it with iOS 16+ API after the app is iOS 16+? `navigationBarHidden` seems to still work for iOS 15 in Xcode 14 beta.

```suggestion
        if #available(iOS 16.0, *) {
            navigationController?.isNavigationBarHidden = true
        }
```